src/traces/scatter/attributes.js

```diff
@@ -686,4 +686,14 @@ module.exports = {
         arrayOk: true,
         description: 'Sets the text font.'
     }),
+    zindex: {
+        valType: 'integer',
+        dflt: 0,
+        editType: 'calc',
```

Comment:
It looks like we don't need to redo the `calc` steps of traces when `zindex` is changing.
For this reason and improve performance in react calls let's change the `editType` to be *plot*. 
```suggestion
        editType: 'plot',
```